### ISSUE TITLE
Command init doesn't create all used folders

### ISSUE THREAD

**Issue by dvidsilva** [gloriajs/gloria#81]
Folders like _sass, _stylus and _data are not created when running the command init.

**Comment by dfrownfelter** [comment]
What about `build` with a config file?

**Comment by dfrownfelter** [comment]
Should we make these default or let people create the folders on their own?

**Comment by dvidsilva** [comment]
yeah that's what i was wondering, but it would be nice to have some defaults so that people know they can use them, if they don't use it then it just gets ignored by the build process.

**Comment by dvidsilva** [comment]
For this, a better way to phrase it would be to clean out the default file and folder structure created when running init.
